lamping.md

```diff
@@ -0,0 +1,2 @@
+As you begin to scroll through your feed on IG and using your 
```

Comment:
It looks like you moved this file to another directory, but now you'll need to delete this version from Git. You can use the `git rm` command to do this. You can also use `git rm` to remove the `.DS_Store` file.